src/task_types/resmoke_config_writer.rs

```diff
@@ -192,27 +133,22 @@ impl WriteConfigActorImpl {
     /// # Arguments
     ///
     /// * `sub_suites` - List of sub-suites to write configuration for.
-    /// * `origin_config` - Configuration to base sub-suite configuration on.
-    /// * `target_name` - Name to base generated file on.
+    /// * `resmoke_config_cache` - Cache to get resmoke suite configurations.
     fn write_sub_suites(
         &self,
         sub_suites: &[SubSuite],
-        origin_config: &ResmokeSuiteConfig,
-        target_name: &str,
+        resmoke_config_cache: &mut ResmokeConfigCache,
     ) -> Result<()> {
         let results: Result<Vec<()>> = sub_suites
             .iter()
+            .filter(|s| s.exclude_test_list.is_none())
```

Comment:
This `exclude_test_list` field is new. Instead of this code trying to create the tests to exclude for _misc suites (which gets complicated for multiversion), the code creating the splits will do it and include it in the `SubSuite` structs. This has the added benefit of letting use filter the sub-suites based on that field.